packages/workflow/src/records/handler/reinstate.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import * as Hapi from '@hapi/hapi'
+import * as z from 'zod'
+import { getToken } from '@workflow/utils/authUtils'
+import { validateRequest } from '@workflow/utils/index'
+import { getRecordById } from '@workflow/records/index'
+import { toReinstated } from '@workflow/records/state-transitions'
+import { sendBundleToHearth } from '@workflow/records/fhir'
+import { indexBundle } from '@workflow/records/search'
+import { SavedBundleEntry, Task } from '@opencrvs/commons/types'
+
+export async function reinstateRecordHandler(
+  request: Hapi.Request,
+  _: Hapi.ResponseToolkit
+) {
```

Comment:
We should use the "createRoute" handler for reinstating, that way we can define which states we are expecting a record to be reinstated to